spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -852,6 +848,11 @@ def then_there_should_be_no_publish_button
 
   alias_method :and_there_should_be_no_publish_button, :then_there_should_be_no_publish_button
 
+  def and_I_cannot_publish_or_schedule_the_step_by_step
+    then_there_should_be_no_publish_button
+    and_there_should_be_no_schedule_button
```

Comment:
There's an alias for this method, `then_there_should_be_no_schedule_button` that should also be removed.

Reply:
Good spot.